docs/source/verdi/verdi_user_guide.rst

```diff
@@ -26,155 +48,108 @@ For example ``verdi process kill --help`` will display:
                          soon as it's scheduled.
     -h, --help           Show this message and exit.
 
-All help strings have the same format and consist of three parts:
+All help strings consist of three parts:
 
-  * A usage line describing how to invoke and the accepted parameters
-  * A description of the commands functionality
+  * A ``Usage:`` line describing how to invoke the command
+  * A description of the command's functionality
   * A list of the available options
 
-The ``[OPTIONS]`` and ``[PROCESSES]...`` tags in the usage description, denote the 'parameters' that the command takes.
-A more detailed description of the available options will be listed after the description of the commands functionality
-The positional arguments will only be described in the command description itself.
-For a more detailed explanation of the difference between options and arguments, see the section about command line :ref:`parameters<cli_parameters>`.
+The ``Usage:`` line encodes information on the command's parameters, e.g.:
 
+ * ``[OPTIONS]``: this command takes one (or more) options
+ * ``PROCESSES``: this command requires a process as a positional argument
```

Comment:
Since the importance here is the distinction of _requires_ and _optional_, maybe highlight these words?